CleanRepo/Program.cs

```diff
@@ -432,6 +422,18 @@ private static void ListOrphanedImages(string inputDirectory, Dictionary<string,
             Console.WriteLine("DONE");
         }
 
+        private static string TryGetFullPath(string path)
+        {
+            try
+            {
+                return Path.GetFullPath(path);
+            }
+            catch
```

Comment:
Do we want to explicitly catch PathTooLongException here and log it to the output so the user is aware that they have some really long paths that a) are being ignored and b) should perhaps be shortened? Perhaps do something similar on line 410 too.

Reply:
Yes, that is a great idea. I updated it to log explicitly for path too long and added the more generic reason if it is something else that we've yet to experience.